charts/dave/charts/backend/templates/deployment.yaml

```diff
@@ -63,10 +63,12 @@ spec:
           envFrom:
             - configMapRef:
                 name: {{ include "daveBackend.fullname" . }}
-            - configMapRef:
-                name: {{ include "daveBackend.fullname" . }}-email
             - secretRef:
+                {{- if .Values.backend.existingSecret }}
+                name: {{ .Values.backend.existingSecret }}  
```

Comment:
The `{{ include "daveBackend.fullname" . }}-secret` is still created by the chart, though. Maybe add a conditional in `secret.yaml` as well?